src/Dscp/DB/SQLite/SchemaInit.hs

```diff
@@ -0,0 +1,16 @@
+
+{-# LANGUAGE QuasiQuotes #-}
+
+module Dscp.DB.SQLite.SchemaInit
+    ( ensureSchemaIsSetUp
```

Comment:
I don't see a reason why `Schema` and `SchemaInit` are two different modules.

Reply:
I will fix this now. After second thought, only the `Schema` module should remain, consuming `SchemaInit`.